README.md

```diff
@@ -203,7 +203,35 @@ COPY bin/my-wag-service /usr/bin/my-wag-service
 
   * If you don't have a context to pass to a Wag function you have two options
     * context.Background() - use this when this is the creator of the request chain, like a test or a top-level service.
-    * context.TODO() - use this when you haven't been passed a context from a caller yet, but you expect the caller to send you one at some point.  
+    * context.TODO() - use this when you haven't been passed a context from a caller yet, but you expect the caller to send you one at some point.
+
+
+### DynamoDB Codegen
+
+  * Wag can auto-generate server code to save models to DynamoDB if you specify the `x-db` extension on a schema:
```

Comment:
Do we imagine other dbs will be supported? Shoul we namespace to DynamoDB (`x-dynamodb`?)

Reply:
My thinking here:
- yes, in the future it may support other db types
- `x-db` top level config should be independent of db-type
- `DynamoDB` top level config contains DynamoDB-specific stuff